scripts/zones/Southern_San_dOria/npcs/Kipopo.lua

```diff
@@ -24,13 +24,13 @@ end;
 function onTrigger(player,npc)
     local sayItWithAHandbag = player:getQuestStatus(CRYSTAL_WAR, dsp.quest.id.crystalWar.SAY_IT_WITH_A_HANDBAG)
     local sayItWithAHandbagCS = player:getVar("sayItWithAHandbagCS")
+    local sayItWithAHandbagBonusCS = player:getVar("sayItWithAHandbagBonusCS")
     local guildMember = isGuildMember(player,7)
     local SkillCap = getCraftSkillCap(player, dsp.skill.LEATHERCRAFT)
     local SkillLevel = player:getSkillLevel(dsp.skill.LEATHERCRAFT)
 
-    if sayItWithAHandbag == QUEST_COMPLETED and sayItWithAHandbagCS == 4 and player:hasItem(19110) then
+    if sayItWithAHandbag == QUEST_COMPLETED and player:hasItem(19110) and sayItWithAHandbagBonusCS == 1 then
```

Comment:
Can drop the `player:hasItem(19110)`. They must have been given it for the quest to be completed, _and_ you want them to be able to get the optional dialogue (more importantly: clear out their SQL var) even if they've tossed the item!

Reply:
I didn't think about that scenario.  Thanks for pointing it out. Will be removing it so player can continue the quest even tho they have tossed the item to clear the var. this has been already added onEventFinish for csid == 914.

Thanks for your feedback 😸